typegraph/typegraph/runtimes/http.py

```diff
@@ -85,6 +85,7 @@ class RESTMat(Materializer):
     content_type: str = field(kw_only=True, default="application/json")
     header_prefix: Optional[str] = "header#"
     query_fields: Optional[Tuple[str, ...]] = field(kw_only=True, default=None)
+    rename_fields: Optional[Tuple[Tuple]] = field(kw_only=True, default=None)
```

Comment:
`Optional[Tuple[Tuple[str, str]]]`?

Reply:
I think a `dict` would be easier to write; and will not produce this kind of mistake: `(("field", "renamed"))`.
You can set `field(converter=frozendict)` to automatically transform it into `frozendict`.

Reply:
You are right, I had errors with a raw dict as it is not hashable so I used tuple as a proof of concept. Thanks, I will try that.